azure-pipelines.yml

```diff
@@ -31,13 +31,21 @@ jobs:
           test_suites:
               all: venv/bin/pytest -n 2 -vvs --reruns 2
 
+    - template: etc/ci/azure-posix.yml
+      parameters:
+          job_name: macos1015_cpython
+          image_name: macos-10.15
+          python_versions: ['3.7', '3.8', '3.9', '3.10', '3.11']
+          test_suites:
+              all: venv/bin/pytest -n 2 -vvs --reruns 2
+
     - template: etc/ci/azure-posix.yml
       parameters:
           job_name: macos11_cpython
           image_name: macos-11
           python_versions: ['3.7', '3.8', '3.9', '3.10', '3.11']
           test_suites:
-              all: venv/bin/pytest -n 2 -vvs --reruns 2
+             all: venv/bin/pytest -n 2 -vvs --reruns 2
```

Comment:
I can see some changes are highlighted here, please remove unrelated changes

Reply:
Done

Reply:
It still shows unrelated changes here

Reply:
I guess its done now, plz have a look @TG1999